紀錄/LeetCode 讀書會第 20 次聚會 2021_01_26.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
# LeetCode 讀書會第 20 次聚會 2021/01/26

leetcode 讀書會通知
1. 項目: 第 20  次聚會
2. 目的: 線上一起寫題目, 由有想法的人帶領, 先解題, 再看該題有趣的解法
3. 時間: 1/26 (二) 20:00 ~ 21:00
4. 地點: google meet 線上 (前 10 分鐘預備鏈接)
5. 解題項目:  [Linked list - Doubly linked list](https://leetcode.com/explore/learn/card/linked-list/210/doubly-linked-list/)

```text
### Doubly Linked list
  1. Medium 707. Design Linked List

### Conclusion
  1.  Easy 21. Merge Two Sorted Lists 
  2. Medium    2. Add Two Numbers 
  3. Medium    430. Flatten a Multilevel Doubly Linked 
                                > List Hint: Try to solve this problem recursively.
  4. Medium    708.  Insert into a Cyclic Sorted List 
                                 >  Hint: A cyclic list here is a linked list that the last node is linked with the first node and form a cycle. Find out the smallest node will help you determine the suitable position.
  5. Medium    138.  Copy List with Random Pointer 
                                  > Hint: Using a hash set maybe is a good idea. Refer to our hash table card if you are not familiar with the hash set.
  6. Medium    61.  Rotate List 
```

6. 共筆: GitHub https://github.com/programmingbookclub/Leetcode-club

```